Auto Steamworks.sln

```diff
@@ -5,8 +5,6 @@ VisualStudioVersion = 16.0.29613.14
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoSteamApp", "AutoSteamApp\AutoSteamApp.csproj", "{480E341A-4063-4DA4-9B77-5443A0B923AF}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HarvestInfo", "HarvestInfo\HarvestInfo.csproj", "{BC797936-68FA-41D3-9DFE-8F7B6BC446D3}"
-EndProject
```

Comment:
Even if the HarvestInfo app is not part of the AutoSteamApp, we can eighter create a new solution or keep it in this one

Reply:
Fixed in newest merge to develop